qi-lib/flow/core/compiler/deforest/cps.rkt

```diff
@@ -7,7 +7,8 @@
                      "syntax.rkt"
                      "../../../extended/util.rkt"
                      syntax/srcloc
-                     racket/syntax-srcloc)
```

Comment:
RE 8bfe50272f5c5e31993c88869987e21dd72bbb7f:

- The subject is too long
- My preference on addressing review feedback is to fixup the relevant commits
with feedback (_e.g._, create commits with `--fixup` and then `rebase
--autosquash` them). You can show us the changes from the rebase by doing
something like `git range-diff main @{u} @` before force-pushing the rebased
branch. This keeps the history in a good state (if desired, new information can always be added to the fixed commit's messages, too).